libvast/src/concept/parseable/vast/expression.cpp

```diff
@@ -44,6 +45,11 @@ static predicate::operand to_field_extractor(std::vector<std::string> xs) {
 }
 
 static predicate::operand to_attr_extractor(std::string x) {
+  if (x == "timestamp") {
+    VAST_WARN("#timestamp queries are deprecated and can be replaced with "
```

Comment:
```suggestion
    VAST_WARN("#timestamp queries are deprecated and should be replaced with "
```

We are going to remove this functionality eventually, so we should encourage the change.